### ISSUE TITLE
Needs inline help for tags and functions

### ISSUE THREAD

**Issue by redtopia** [ilich/vscode-coldfusion#12]
It would be helpful to have an easy way to access inline help for tags and functions like the CFML Package for Sublime Text 3 plugin (https://github.com/jcberquist/sublimetext-cfml). This screenshot shows what happens when I hover over a function, which has made a big difference in my workflow:

![image](https://cloud.githubusercontent.com/assets/812777/25635510/a201ce30-2f3b-11e7-948f-996c81b3cb0a.png)

**Comment by shakeelosmani** [comment]
I agree this would be awesome, in fact with this you can go commercial 💃

**Comment by KamasamaK** [comment]
It's not as robust as Sublime's, but I took a shot at this. It doesn't check the syntax and only works with what's on CFDocs.
